internal/controller/config/templates/istio/authconfig.yaml.tmpl

```diff
@@ -35,9 +35,9 @@ spec:
     k8s-rbac:
       kubernetesSubjectAccessReview:
         user:
-          selector: auth.identity.metadata.annotations.userid
-        groups:
-          - {{.Name}}-users
+          selector: auth.identity.user.username
+        authorizationGroups:
+          selector: auth.identity.user.groups
```

Comment:
Does this behave as a `OR` or as an `AND`? i.e. will a user who is allowed but is not part of an allowed group be considered allowed?

Reply:
The selector grabs the list of all groups that the user is part of from the K8s TokenReview. So, it actually dynamically tests all user/serviceaccount groups rolebdindings as well as the user/SA's own rolebindings. 

So, users can either bind to individual accounts or groups.